README.md

```diff
@@ -9,17 +9,23 @@
 
 Maintainer: [Michal Kruczek](https://github.com/partikus)
 
-This stack uses: [nginx](https://hub.docker.com/_/nginx/) (lastest), [php-fpm](https://hub.docker.com/_/php/) (5.6, 7.0), [mysql](https://hub.docker.com/_/mysql/) (5.7).
+This stack uses: [nginx](https://hub.docker.com/_/nginx/) (lastest), [php-fpm](https://hub.docker.com/_/php/) (5.6, 7.1, 7.0), [mysql](https://hub.docker.com/_/mysql/) (5.7).
 
 This skeleton can be used to developing app locally or to running tests in Jenkins etc.
 
 1. Build docker images running ``./docker.sh build-images``. Image versioning is based on `version` from `composer.json`
-2. Running local dev env: ``./docker.sh run`` Domain is set based on ``.project_name``, by default ``http://demo.dev:8000``
-3. Running tests on PHP 7.1: ``./docker.sh build``
-4. Running tests on PHP 7.1 with CodeCoverage(include xdebug): ``./docker.sh build-coverage``
-5. Running tests on PHP 7.0: ``./docker.sh build-7``
-6. Running tests on PHP 7.0 with CodeCoverage(include xdebug): ``./docker.sh build-coverage-7``
-7. Running tests on PHP 5.6: ``./docker.sh build-56``
+2. Running local dev env: ``./docker.sh run`` Domain is set based on ``.composer.json`` ``name`` property without ``/`` sign, by default ``http://pgsdemo.dev:8000`` PHP 7.1
```

Comment:
maybe would better to describe the main commands overview as a table e.g.:
<table>
<tr>
<td><code>build</code> <br> <code>build-71</code></td>
<td>An argument is running builds under php 7.1 version<br> <code>build</code> it's just an alias to <code>build-71</code> and always refer to the newest version</td>
</tr>
<tr>
<td><code>build-70</code></td>
<td>An argument is running builds under php 7.0 version</td>
</tr>
</table>